optashift-employee-rostering-server/src/main/java/org/optaplanner/openshift/employeerostering/server/common/jaxrs/OptaShiftObjectMapperResolver.java

```diff
@@ -16,25 +16,28 @@
 
 package org.optaplanner.openshift.employeerostering.server.common.jaxrs;
 
-import javax.persistence.EntityNotFoundException;
-import javax.ws.rs.core.MediaType;
-import javax.ws.rs.core.Response;
-import javax.ws.rs.core.Response.Status;
 import javax.ws.rs.ext.ContextResolver;
-import javax.ws.rs.ext.ExceptionMapper;
 import javax.ws.rs.ext.Provider;
 
+import com.fasterxml.jackson.databind.DeserializationFeature;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
 import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import org.optaplanner.persistence.jackson.api.OptaPlannerJacksonModule;
 
 @Provider
 public class OptaShiftObjectMapperResolver implements ContextResolver<ObjectMapper> {
 
     private final ObjectMapper objectMapper;
 
     public OptaShiftObjectMapperResolver() {
+        // JavaTimeModule defaults date format to ISO, so we don't need to deal with the headache of setting
+        // a universal date format that is ISO and includes an offset
```

Comment:
This comment is confusing to me, let's remove it in favor of the other comment I am suggesting below.
The general idea of referring to the toString() is "I thrust the java.time creators to have chosen the best standarized OffsetDateTime format for it's toString().". On top of that, it adds an output example to clarify what we expect to see.